app/views/developers/show.html.erb

```diff
@@ -87,7 +85,7 @@
             LinkedIn
           </dt>
           <dd class="mt-1 text-sm text-gray-900">
-            <%= link_to @developer.linkedin , "https://www.linkedin.com/in/#{@developer.linkedin}/", target: "_blank", class: "hover:text-gray-600" %>
+            <%= link_to @developer.linkedin , developer_link_path(@developer, :linkedin), target: "_blank", class: "hover:text-gray-600" %>
```

Comment:
I like this approach! I was thinking about this issue when you first added it and this is about how I pictured it working

Reply:
Thanks! It gives us fine grain access control without having to change the markup of the `#show` page at all. Ideally, we can start doing analytics server side, too!